index.js

```diff
@@ -325,8 +325,8 @@ function parse (args, opts) {
       i = ii
       argsToSet.push(args[ii])
     }
-    if (multipleArrayFlag && !configuration['flatten-duplicate-arrays']) {
-      setArg(key, argsToSet)
+    if (multipleArrayFlag) {
+      setArg(key, argsToSet.map(arg => processValue(key, arg)))
```

Comment:
We want to maintain Node 0.10 support for now (until early next year), so the arrow function needs to be replaced with a plain old function.